### DISCUSSION TITLE
How can I hide this error?

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#452] [Q&A]
![image](https://user-images.githubusercontent.com/68460690/203858208-bba6bf2c-6e8e-458a-b911-c4d46bfd24fa.png)
I catch the exception, but it still appears it seems.
See https://github.com/internetarchive/wcdimportbot/blob/017d98fae901a0738f5961d9967935a3c67450d3/src/models/wikibase/crud/create.py#L39

**Comment by LeMyst** [comment]
Hello Dennis,

If you use the default level for logging (WARNING) the log.exception in baseentity:https://github.com/LeMyst/WikibaseIntegrator/blob/519deef572fe149293bb20c41024338e19ee6214/wikibaseintegrator/entities/baseentity.py#L257 
will always output a stacktrace...

I don't know what I should do with this.
Does I remove all log.exception from WikibaseIntegrator and let the users to catch the exception? (Because WBI is a library, and it's not it work to catch and print message?)

**Reply by dpriskorn** [reply]
I use level ERROR in production but it outputs the stack trace no matter what I do.

**Reply by dpriskorn** [reply]
> Does I remove all log.exception from WikibaseIntegrator and let the users to catch the exception? (Because WBI is a library, and it's not it work to catch and print message?)

Yes that sounds nice. Then I can decide downstream what gets printed and handled, reprised, etc.
